### DISCUSSION TITLE
Daphne on Rg280v

### DISCUSSION THREAD

**Discussion by SlowFat** [eduardofilo/RG350_adam_image#228] [Q&A]
I can't for the life of me get Daphne to work on the rg280v on the latest version of Adam image. Any insight on how to properly set this up? I tried setting it up as demonstrated in this video but no luck. https://youtu.be/ch3NP0tvZdA

Thank you for the help!

**Comment by eduardofilo** [comment]
Well, that guide is for RG351. In Adam all you need to do is to put the files in the correct format in the roms/DAPHNE directory of the external card.
I have a guide, although a bit outdated I think it is still valid. It is in spanish but surely you can follow it with a translator.
https://apuntes.eduardofilo.es/2020-05-04-rg350_daphne.html
